Pipfile

```diff
@@ -26,10 +26,10 @@ bcrypt = "==3.0.0"
 boto3 = "==1.4.5"
 celery = "==4.3.0"
 ciso8601 = "==1.0.3"
-Django = "==1.11.28"
+Django = "==2.2"
 django-ace-overlay = "==0.8.0"
 django-allauth = "==0.32.0"
-django-bitfield = "==1.9.3"
+django-bitfield = "==2.0.1"
```

Comment:
make sure you change this django-bitfield requirement in `requirements/default.txt` as well

Reply:
There were some others that weren't aligned, I think now everything's lined up between Pipfile & requirements